components/pages/WalletSettings/OtherWallet.tsx

```diff
@@ -25,8 +25,8 @@ export function OtherWallet({
 
   const text =
     status === "delegate"
-      ? "Explanation of secondary wallet when secondary is delegate."
-      : "Explanation of secondary wallet when secondary is delegator.";
+      ? "Explanation of secondary wallet when secondary is delegate. Do this."
+      : "Explanation of secondary wallet when secondary is delegator. Do this.";
```

Comment:
generally in code when something needs to be done like this you use `TODO: "Explanation of secondary wallet when secondary is delegate"`.  this makes it easy to search a codebase to find any todos